tests/Session/SessionTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Tests\Session;
+
+use Yiisoft\Yii\Web\Session\Session;
+use PHPUnit\Framework\TestCase;
+
+class SessionTest extends TestCase
+{
+    public function tearDown()
+    {
+        @session_destroy();
+    }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testGetAndSet()
+    {
+        $session = new Session();
+        $session->set('get', 'set');
```

Comment:
Let's make these names a bit more obvious.

```suggestion
        $session->set('key_get', 'value');
```